### ISSUE TITLE
Event sent to lambda is a string

### ISSUE THREAD

**Issue by ibabicuy** [FidelLimited/serverless-plugin-warmup#102]
up to commit 6aa041cd4e097a656b94683cf4ab2d41f9612c2f, the event sent to the lambdas is a string, which makes the 
```
if event.get('source', None) == 'serverless-plugin-warmup':
```
Escape fail, as now event is a string

**Comment by juanjoDiaz** [comment]
Hi @ibabicuy,

I'm not sure I understand the problem. The link you mention doesn't seem to touch the event at all (only some docs).

Can you clarify a bit more?

**Comment by juanjoDiaz** [comment]
I think that this was fixed by #114. (Not released yet)

**Comment by juanjoDiaz** [comment]
Should be fixed after 4.3.X.

Feel free reopen if it's not.
